rta/__main__.py

```diff
@@ -37,25 +38,35 @@ def run_all():
 
 def run(ttp_name: str, *args):
     """Run all RTAs compatible with OS."""
-    if ttp_name not in get_ttp_names():
+    if ttp_name not in get_available_tests().keys():
         raise ValueError(f"Unknown RTA {ttp_name}")
 
     module = importlib.import_module("rta." + ttp_name)
     return module.main(*args)
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
     parser = argparse.ArgumentParser("rta")
-    parser.add_argument("--ttp-name")
+    parser.add_argument("-n", "--name", dest="name", nargs='+',
+                        help="Name(s) of test(s) to execute. E.g. bitsadmin_execution adobe_hijack")
+    parser.add_argument("-l", "--list", dest="list", action="store_true", help="Print a list of available tests")
+    parser.add_argument("-o", "--os-filter", dest="os_filter", default="all", choices=RTA_PLATFORM_TYPES,
+                        help="Filter rule coverage summary by OS. (E.g. windows) Default: all",)
     parser.add_argument("--run-all", action="store_true")
     parser.add_argument("--delay", type=int, help="For run-all, the delay between executions")
     parsed_args, remaining = parser.parse_known_args()
 
-    if parsed_args.ttp_name and parsed_args.run_all:
-        raise ValueError(f"Pass --ttp-name or --run-all, not both")
-
-    if parsed_args.run_all:
+    if parsed_args.name:
+        if parsed_args.run_all:
+            raise ValueError(f"Pass --ttp-name or --run-all, not both")
+        else:
+            for rta_test in parsed_args.name:
+                rta_name = Path(rta_test).stem
```

Comment:
Is the error specific/obvious enough for a missing file or should it be wrapped

Reply:
Makes sense to me, but I did clean this up a bit.